packages/@ember/-internals/glimmer/tests/integration/application/engine-test.js

```diff
@@ -168,7 +172,9 @@ moduleFor(
         'engine:blog',
         Engine.extend({
           init() {
-            this._super(...arguments);
+            ignoreDeprecation(() => {
```

Comment:
Same as above. Will only comment on these two instances, but I think most (possibly all?) of the `ignoreDeprecation` calls (in this test file at least) should be removed and we should figure out how to avoid the deprecation in the tests.

Reply:
I think I need to refactor all of the engines tests to use the ModuleBasedTestResolver instead of the DefaultResolver.

Reply:
Ya, that sounds right